syntax = "proto3";
option objc_class_prefix = "HLW";
package helloworld;
service InferenceService {
  rpc Inference (stream InferenceRequest) returns (stream InferenceResponse);
}
message InferenceRequest {
  repeated int64 shape = 1;
  repeated float data = 2;
}
message InferenceResponse{
  repeated float data = 1;
}

message endMessageRequest{

}

message endMessageResponse{
  float data = 1;
}

service InferenceService2 {
  rpc Inference2 (InferenceRequest) returns (InferenceResponse);
  rpc end (endMessageRequest) returns (endMessageResponse);
}